pkg/controller/containerjfr/containerjfr_controller.go

```diff
@@ -95,38 +95,17 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	}
 
 	// Watch for changes to secondary resource Deployements and requeue the owner ContainerJFR
```

Comment:
I suppose now we should change this comment to just `secondary resources` now that it's more than just Deployments.

Reply:
Done.